frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -312,6 +319,52 @@ const HostSummary = ({
     );
   };
 
+  const renderOperatingSystemSummary = () => {
+    // No tooltip if minimum version is not set, including all Windows, Linux, ChromeOS operating systems
+    if (!osUpdatesData?.minimum_version) {
+      return (
+        <DataSet title="Operating system" value={summaryData.os_version} />
+      );
+    }
+
+    const tooltipMessage = (osVersion: string, minimumOsVersion: string) => {
```

Comment:
Can we unnest this definition and name it `getTooltipMessage`?

Reply:
ooo good idea, I see you made one already `getHostDiskEncryptionTooltipMessage` on the top of the file.